subsys/bootloader/image/CMakeLists.txt

```diff
@@ -4,15 +4,14 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-set(KERNEL_ELF kernel_elf) # TODO multi_image
 zephyr_include_directories(${CMAKE_CURRENT_LIST_DIR}/../include)
 if (NOT DEFINED CONFIG_IS_SECURE_BOOTLOADER)
   add_subdirectory(bl_crypto_client)
 endif()
 set(NRF_BOOTLOADER_SCRIPTS ${NRF_DIR}/scripts/bootloader)
 set(PROVISION_HEX_NAME     provision_data.hex)
 set(PROVISION_HEX          ${PROJECT_BINARY_DIR}/${PROVISION_HEX_NAME})
-set(SIGNED_KERNEL_HEX      ${CMAKE_BINARY_DIR}/signed_by_b0_${KERNEL_HEX_NAME})
+set(SIGNED_KERNEL_HEX      ${PROJECT_BINARY_DIR}/signed_by_b0_${KERNEL_HEX_NAME})
```

Comment:
Not sure if this change should be in this commit but doesn't it make more sense to put the signed hex file in the build directory inside zehyr/ folder where the other hex files are instead of outside?

Reply:
Think this way is better